include/zephyr/pm/policy.h

```diff
@@ -278,6 +311,18 @@ static inline void pm_policy_event_unregister(struct pm_policy_event *evt)
 {
 	ARG_UNUSED(evt);
 }
+
+void pm_policy_next_event_subscribe(struct pm_policy_event_subscription *req,
```

Comment:
`static inline` otherwise each object will have its own implementation of this function and the linker will complain